### ISSUE TITLE
WebView has been removed from ReactNative

### ISSUE THREAD

**Issue by srhtylmz19** [wix/react-native-zss-rich-text-editor#174]
i got this error on simulator but i dont use webview at all

**Comment by export-mike** [comment]
its used by react-native-zss-rich-text-editor.

**Comment by srhtylmz19** [comment]
thats why i typed under this repo.

**Comment by MaxInMoon** [comment]
For me, the error occurs on an Android real device, and not in the iOS simulator

**Comment by export-mike** [comment]
https://github.com/export-mike/react-native-zss-rich-text-editor/tree/ios13 removing all usage of UIWebView 
here its working but having issues with the toolbar being behind the "auto-suggest"

**Comment by zelin-zx** [comment]
我也遇到了相同的问题,你解决了吗

**Comment by export-mike** [comment]
yes see this link https://github.com/export-mike/react-native-zss-rich-text-editor/tree/ios13 it has one small issue.

**Comment by jb-** [comment]
I replaced react-native/webview with react-native-webview. 
https://github.com/jb-/react-native-zss-rich-text-editor

**Comment by HemantRMali** [comment]
pod intall 
 It should install react-native-webview  (version 10+)

Then error has gone. showing toolbar, but it does not show <RichTextEditor>

**Comment by nasy** [comment]
Just installed this library and I am getting this error "Webview has been removed from React Native". Did you guys manage to solve the problem?
